src/styles/styles.scss

```diff
@@ -37,6 +36,62 @@ html, body {
   height: 100%;
 }
 
+.index_bottom_container {
```

Comment:
please can you try to keep PRs tidy and not also commit changes from other branches? Could you try to remove any changes in this PR that are already in some other PR?